docs/source/deployment.rst

```diff
@@ -8,43 +8,91 @@ This will be run automatically when you attempt to commit code but if you want t
 
 .. code-block:: console
 
-    poetry run pre-commit run --all-files
+    $ poetry run pre-commit run --all-files
 
 This app is set up to use circleci, but could be extended to any build process. Circle will automatically run:
 
 .. code-block:: console
 
-    poetry run pre-commit run --all-files
-    poetry run playwright install
-    poetry run coverage run --source='.' -m pytest
+    $ poetry run pre-commit run --all-files
+    $ poetry run playwright install
+    $ poetry run coverage run --source='.' -m pytest
 
 
 Compiling Process
 -----------------
 
-Vite and Entry Points
-*********************
+Vite and Assets
+***************
 
 Vite must be made aware of all of the frontend assets in order to properly bundle and run them.
 
 To start, it's worth noting that Vite supports both typescript and javascript files out of the box, so there's flexibility for you to
 build how you see fit.
 
-When you're ready to import your .js or .ts files into Vite, you'll need an entrypoint.
-In the ``vite.config.js`` file, there is a single input in ``rollupOptions`` named ``main``.
-This is set to the ``main.ts`` file in the frontend directory, and that file serves as the central entrypoint for all of the assets Vite will bundle.
+When you're ready to import your .js or .ts files into Vite, you'll need an asset.
+In the ``vite.config.js`` file, there are three assets (called inputs in ``rollupOptions``):
 
-In the main.ts file, Alpine.js and htmx have already been imported, but any additional frontend packages/frameworks should be imported using
+* ``main``: set to ``fontend/js/main.ts``, acts as the central entrypoint for all JS/TS assets.
```

Comment:
This will probably need to be updated for the frontend reorg pr

Reply:
I'll add a comment here to note this.